vue-vuetify/src/util/composition.ts

```diff
@@ -87,6 +88,10 @@ export const useVuetifyControl = <
   const appliedOptions = useControlAppliedOptions(input);
   const isFocused = ref(false);
 
+  const filteredErrors = computed(() => {
+    return (touched.value || !input.control.value.config.enableFilterErrorsBeforeTouch)? input.control.value.errors : '';
```

Comment:
We could use the `appliedOptions` instead of the `config` so that the touched behavior can be enabled/disabled for specific fields via the ui schema.